File: breakingtherecords.js
function getRecord(s){
    var low = s[0];
    var high = s[0];
    var breakLow = 0;
    var breakHigh = 0;
    for(var i = 1; i < s.length; i++){
        if(s[i] > high){high = s[i]; breakHigh++}
        else if(s[i] < low){low = s[i]; breakLow++};
    }
    console.log(breakHigh + ' ' + breakLow);
}

File: breakingtherecords.rb
#!/bin/ruby

def getRecord(s)
    
    low = s[0]
    high = s[0]
    break_low = 0
    break_high = 0
    
    for i in s do
    if i > high
        high = i
        break_high = break_high + 1
    
    elsif i < low
        low = i
        break_low = break_low + 1
        
    end
    end  
    
    puts"#{break_high} #{break_low}"
    
end

n = gets.strip.to_i
s = gets.strip
s = s.split(' ').map(&:to_i)
result = getRecord(s)
print result.join(" ")
